app/src/main/java/com/kabouzeid/gramophone/util/PreferenceUtil.java

```diff
@@ -311,6 +312,37 @@ private long getCutoffTimeMillis(final String cutoff) {
         return (System.currentTimeMillis() - interval);
     }
 
+    public String getLastAddedCutoffText(Context context) {
```

Comment:
I feel like this stuff doesn't quite belong in `PreferenceUtil`. Could you move it to its own util class, perhaps?